whctools/models.py

```diff
@@ -62,6 +66,32 @@ class Meta:
         ordering = ["eve_character__character_name"]
         verbose_name_plural = "Applications"
 
+    def get_main_character(self):
+        return get_main_character_from_evecharacter(self.eve_character)
```

Comment:
See above about AA 4.0.
Since this is in its own method, you could make a workaround here for now. Again its your choice